### ISSUE TITLE
{Linked:Bug3397200}[Adaptive Cards > Designer] - Screen Reader does not announce the information as Empty Adaptive card.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8825]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 23H2 (OS Build 26040.1000)
Browser Version: Version 122.0.2365.3 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**
1. Open the above URL on Edge Dev URL.
2. Press tab key and navigate to the 'New Card' and press enter key to activate it.
3. New Card dialog will be appeared.
4. Navigate to the 'Blank Card' using tab key and press enter key to activate it.
5. Navigate to the 'Empty Adaptive Card' and observe the issue.

**Actual Result:**
Screen Reader does not announce the information as Empty Adaptive card, it announces as 'Card preview region Group' but does not announce that it is empty,
Also, Screen reader is announcing the infomration as Group only.

**Expected Result:** 
Screen Reader must announce the complete information as Card preview region group adaptive card empty clickable.
Also, when there is any label present then it should announce the label infomration as well.

**Observation:**
Issue is also repro with NVDA.

**Note:**
1. While navigating in backward navigation it is announcing the information as 'Blank'.
2. Same issue is observed when user is navigating to controls present under Editor section. Screen Reader is conveying only as  "Group" and "Double click to edit group".

**User Experience:**
Users who rely on screen readers will not get the proper information that the card is empty, and no action will occur when they press enter and so they may get confused and face trouble in accessing the UI properly.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Screen Reader does not announce the information as Empty Adaptive card](https://github.com/microsoft/AdaptiveCards/assets/95913748/9d65778b-6d03-48f8-a2b4-617415a17433)

https://github.com/microsoft/AdaptiveCards/assets/95913748/c5a277df-7bb2-4c04-b094-dc80d1780385

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is originally tracked with below URL.

https://dev.azure.com/domoreexp/MSTeams/_workitems/edit/3397200

**Comment by vagpt** [comment]
Hi @anna-dingler,

Any update about this bug?

**Comment by anna-dingler** [comment]
Hi @vagpt,

I do not have any updates on this bug. At this time, my team doesn't have the bandwidth to resolve this issue ahead of schedule. It is currently budgeted for our April sprint.

Thanks,
Anna

**Comment by vagpt** [comment]
Hi @anna-dingler, 

This issue is still repro at our end on the environment below, hence reactivating this issue.

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 24H2 (OS Build 26090.1000)
Browser Version: Version 126.0.2566.1 (Official build) dev (64-bit)
Screen Reader: Narrator

**Attachment:**

https://github.com/microsoft/AdaptiveCards/assets/95913748/104af7ce-9df8-4b5c-82c5-cdae1462509c

**Comment by anna-dingler** [comment]
Hi @vagpt, this fix is pending release on our site. I'll add the appropriate tag - apologies for the confusion.

**Comment by vagpt** [comment]
Hi @anna-dingler,

Hope you are doing great.

As this issue is still repro'ing at our end so could you please let us know by when the changes will be merged on the site.
